src/components/SortPanel/SortPanel.js

```diff
@@ -67,3 +65,26 @@ export default function SortPanel() {
     </div>
   );
 }
+
+function mapStateToProps(state) {
+  const {moviesList: {
+    searchParams,
+  }} = state;
+
+  return {
+    searchParams,
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    fetchMovies: (params) => {dispatch(fetchMovies(params))},
+  }
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(SortPanel);
```

Comment:
You can use short syntax without mapDispatchToProps, like this:
`export default connect(mapStateToProps, { fetchMovies })(SortPanel);`

Reply:
done